cartesian_impedance_msgs/srv/ConfigureCartesianImpedance.srv

```diff
@@ -0,0 +1,4 @@
+cartesian_impedance_msgs/SetCartesianImpedance cartImpedanceMode
+---
+bool success
+string error
```

Comment:
I prefer error enumerations (like those in MoveIt).  It's easier to programmatically respond to enums and it allows for agnostic error recovery.